source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -25,6 +25,34 @@ Common::Redis::RespValuePtr Utility::makeError(const std::string& error) {
   return response;
 }
 
+namespace {
+auto redirectionArgsInvalid = [](const Common::Redis::RespValue* original_request,
```

Comment:
nit: instead of making this a lambda can we just make this a standard anonymous function? Can you also add some doc comments since the parameters are somewhat complex wrt to in/out, etc.

Reply:
done